static/js/ui_init.js

```diff
@@ -464,6 +473,13 @@ export function initialize_everything() {
     const i18n_params = pop_fields("language_list");
 
     realm_logo.rerender();
+
+    // We initialize the sidebars at the top of other modules
+    // to build the page layout early so that it's rendered content
+    // can be used by other modules.
+    // For example- Populating the stream lists/populating private messages.
+    initialize_left_sidebar();
```

Comment:
I rewrote the comments, since this should be next to `compose` and share its comment.